modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-sagemaker.adoc

```diff
@@ -0,0 +1,207 @@
+= Run Teradata Jupyter Notebook Demos for VantageCloud Lake in SageMaker
+:experimental:
+:page-author: Daniel Herrera
+:page-email: daniel.herrera2@teradata.com
+:page-revdate: January 16th, 2024
+:description: Run Teradata Jupyter Notebook Demos for VantageCloud Lake in SageMaker
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, cloud computing, machine learning, sagemaker, vantagecloud, vantagecloud lake, lake
+:dir: sagemaker-guide
+
+== Overview
+Due to their flexibility, scalability, and user-friendly configuration, cloud-based artificial intelligence and machine learning platforms are a staple in the toolkit of many data science teams. This guide details the process for running the https://github.com/Teradata/lake-demos[Teradata Jupyter Notebook Demos for Vantage Cloud Lake], on SageMaker, the AI/ML platform from Amazon AWS.
+
+== Prerequisites
+* https://downloads.teradata.com/download/tools/vantage-modules-for-jupyter[Teradata modules for Jupyter]
+* AWS account with access to S3 and SageMaker.
+* https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[Access to a VantageCloud Lake environment]
+
+== AWS environment set-up
+In this section we will cover in detail each of the steps below:
+* Upload the Teradata modules for Jupyter to an S3 bucket
+* Create an IAM role for your Jupyter notebook instance
+* Create a lifecycle configuration for your Jupyter notebook instance
+* Create Jupyter notebook instance
+* Find the IP CIDR of your Jupyter notebook instance
+
+=== Upload the Teradata modules for Jupyter to an S3 bucket
+* On AWS S3 create a bucket and keep note of the assigned name
+* Default options are appropiate for this bucket
+* In the created bucket upload the Teradata modules for Jupyter +
```

Comment:
I think we need to make it explicit where to get the files from, either here or in the prereqs section. Basically, instead of only providing a link, we need to tell them to download and inform them that the website requires registration.

Reply:
Solved 
![image](https://github.com/Teradata/quickstarts/assets/29882294/e7bf0362-c72e-430f-a2b9-64076f4452b4)